src/main/java/org/openhds/service/impl/census/IndividualService.java

```diff
@@ -39,9 +71,45 @@ public Individual recordIndividual(Individual individual,
                                        String membershipUuid,
                                        String relationshipUuid,
                                        String fieldWorkerId) {
+
         individual.setCollectedBy(fieldWorkerService.findOrMakePlaceHolder(fieldWorkerId));
 
-        //TODO: Handle side effect creation for things like socialgroup and membership etc.
+        //Create the individual first, so that everything after is guaranteed to point at a real entity
+        individual = createOrUpdate(individual);
+
+        //TODO: Is the socialgroup created here?
+        //TODO: Do we make two paths depending on if it's a member or head of household?
+        SocialGroup socialGroup = socialGroupService.findOrMakePlaceHolder(socialGroupUuid);
+        Location location = locationService.findOrMakePlaceHolder(locationUuid);
+
+        //TODO: How are we going to start using project codes?
```

Comment:
At some point I think you are going to add validations based on project codes.

Once the validation is in place, we will have to update our tests and side-effects to use valid codes.